src/data/timer.py

```diff
@@ -75,7 +78,7 @@ def order(self) -> int | None:
     @cached_property
     def round(self) -> int:
         try:
-            return max(int(self.uniq_id), 0)
+            return max(int(self.uniq_id or 0), 0)
```

Comment:
weird typing, I think it should be
```suggestion
            return max(int(self.uniq_id) or 0, 0)
```

Reply:
`max(int(self.uniq_id) or 0, 0)` fails since int can't be passed None.

Reply:
Alright then:
```suggestion
            return max(int(self.uniq_id or '0'), 0)
```
Ugly, but it's consistent (not that I recommend it)